pages/api/gifts/index.ts

```diff
@@ -40,17 +54,29 @@ async function handleGET(req: NextApiRequest, res: NextApiResponse<Gift[]>) {
       updatedAt: true,
       uuid: true,
     },
+    where: {
+      userUUID: userData.uuid,
+    },
   });
 
   return res.status(200).json(gifts);
 }
 
-async function handlePOST(req: NextApiRequest, res: NextApiResponse<Gift>) {
+async function handlePOST(
+  req: NextApiRequest,
+  res: NextApiResponse<Gift>,
+  userData: LuciaUser,
+) {
   const giftData = req.body as CreateGift;
   const addedGift = await prisma.gift.create({
     data: {
       gift: giftData.gift,
       receiver: giftData.receiver,
+      user: {
+        connect: {
+          uuid: userData.uuid,
+        },
+      },
```

Comment:
Olisiko yksinkertaisempi tehdä ilman connectia?
```suggestion
      userUUID: userData.uuid,      
```